docs/development/docguide.rst

```diff
@@ -43,7 +43,13 @@ the standard Astropy docstring format.
 
 * All documentation should be written using the `Sphinx`_
   documentation tool.
-  
+
+  + cross-reference links (``:ref:``, ``:doc:``, etc.), **including internal
+    Astropy links**, should use the `intersphinx format
+    <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_.
+    Note that in Astropy these internal links are to the current build, while
```

Comment:
I think this is not super helpful, maybe add an example? Also, I thought this was required only for the docstrings, not for "all documentation" (which the bullet is about), so I think you need it to one of the following bullet points (maybe the `numpydoc` one? Or just its own bullet point "References in docstrings ..."

Reply:
Done! and noted 'astropy-dev' links to the dev version of astropy.